packages/docs/components/Playground.tsx

```diff
@@ -24,7 +24,7 @@ function Playground({ mode }: {mode?: 'js'|'node'}) {
         // TODO: avoid re-creating the client on every render
         // SyntaxError: Cannot use import statement outside a module
         // clientv2={useCreation(() => new JsClientV2(), [])}
-        clientv2={new JsClientV2()}
+        client={new JsClientV2()}
```

Comment:
Good👍 when appropriate, also rename JsClientV2 -> JsClient

Reply:
to avoid naming conflicts and confusion with an existing `JsClient` in the project, I suggest renaming `JsClientV2` to `WorkspaceApiClient`. What are your thoughts on this proposal?

![image](https://github.com/user-attachments/assets/27cbd4f4-087c-4af7-b2a8-6b64fd6f0f4f)

Reply:
Good idea!